src/catch2/internal/catch_console_colour.cpp

```diff
@@ -22,6 +22,7 @@
 #include <catch2/internal/catch_windows_h_proxy.hpp>
 
 #include <ostream>
+#include <iomanip>
```

Comment:
```suggestion
```
this include is not needed for `std::flush`

Reply:
Removed.